10up/animation-best-practices
# Refactor

### Description of the Change

Moves the repo away from using Docusaurus to using our own tools.
- Uses 10up Toolkit
- Compiles all examples into singular HTML files with inline styles and scripts
- Adds a workflow for automated deploys of singular files
- Removes extra content like documentation, guidelines, etc. and now only tracks example code

### How to test the Change

1. Checkout locally
2. Run `npm install`
3. Run `npm run build`
4. Verify example files are compiled to single HTML documents

### Changelog Entry

See above.


### Credits



### Checklist:
